tests/test_validator_integration.py

```
@@ -293,11 +314,13 @@ def test_random_drop_removed_from_schema(self):
         assert "random_drop" not in forwarded
 
     def test_run_name_forwarded(self):
```

Ruff (0.15.1)

[error] Redefinition of unused `test_run_name_forwarded` from line 291: `test_run_name_forwarded` redefined here

(F811)